cmd/clusterctl/internal/util/objs.go

```diff
@@ -105,42 +120,87 @@ func FixImages(objs []unstructured.Unstructured, alterImageFunc func(image strin
 	// look for resources of kind Deployment and alter the image
```

Comment:
I might have completely missed it, but where is this function called? I see changes to this function, but cannot see where we're calling it

Reply:
This is called in cert_manager.go and components.go